### DISCUSSION TITLE
skoala/registry/managed/servicelist/check-details/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1574] [Announcements]
# skoala/registry/managed/servicelist/check-details/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/registry/managed/servicelist/check-details/

**Comment by liyuerich** [comment]
接口列表和元数据能否给个example？
